### DISCUSSION TITLE
Database updates not active

### DISCUSSION THREAD

**Discussion by arley9107** [centreon-archive/centreon-archived#9652] [Q&A]
![Captura](https://user-images.githubusercontent.com/81997474/113627868-b934c180-9629-11eb-9b37-d826fa93c75b.JPG)

Buenas tardes
Solicito ayuda de como activo Database updates not active en centreon

**Comment by lpinsivy** [comment]
Hi @arley9107 , can you check in the /var/log/centreon-broker/*.log file if you have errors with DB connection?
